### ISSUE TITLE
wan2.2 训练t2vlora

### ISSUE THREAD

**Issue by TTTTong777** [kohya-ss/musubi-tuner#810]
我使用20张图训练，配置如下：
[general]
resolution = [768, 768]
caption_extension = ".txt"
batch_size = 1
enable_bucket = true
bucket_no_upscale = false

[[datasets]]
image_directory = "/root/autodl-tmp/musubi-tuner/dataset/lrt/lrt_test_v4"
cache_directory = "/root/autodl-tmp/musubi-tuner/dataset/lrt/lrt_test_v4_cache_768"
num_repeats = 5

训练参数如下：
python src/musubi_tuner/wan_cache_latents.py \
    --dataset_config /root/autodl-tmp/musubi-tuner/config/lrt_test4.toml \
    --vae /root/autodl-tmp/musubi-tuner/model/vae/wan_2.1_vae.safetensors \
    --clip /root/autodl-tmp/musubi-tuner/model/clip/models_clip_open-clip-xlm-roberta-large-vit-huge-14.pth
python src/musubi_tuner/wan_cache_text_encoder_outputs.py \
    --dataset_config /root/autodl-tmp/musubi-tuner/config/lrt_test4.toml \
    --t5 /root/autodl-tmp/musubi-tuner/model/clip/models_t5_umt5-xxl-enc-bf16.pth \
    --batch_size 16 

CUDA_VISIBLE_DEVICES=0 python src/musubi_tuner/wan_train_network.py \
    --task t2v-A14B \
    --dit /root/autodl-tmp/musubi-tuner/model/wan_model/wan2.2_t2v_high_noise_14B_fp16.safetensors \
    --dataset_config /root/autodl-tmp/musubi-tuner/config/lrt_test4.toml --sdpa --mixed_precision fp16 \
    --optimizer_type adamw8bit --learning_rate 2e-4 \
    --max_data_loader_n_workers 2 --gradient_checkpointing \
    --network_module networks.lora_wan --network_dim 32 \
    --timestep_sampling shift --blocks_to_swap 35 --discrete_flow_shift 8.0 \
    --max_train_epochs 300 --save_every_n_epochs 20 --seed 4332 --min_timestep 875 --max_timestep 1000 --preserve_distribution_shape \
    --output_dir /root/autodl-tmp/musubi-tuner/output/lrt_test_v5 --output_name lrt_test_v5-lora

但是最后的工作流测试结果并没有很好的效果，生成的视频中人物与数据集根本不像，是需要调整哪里吗
